server/src/main/java/io/druid/server/QueryResource.java

```diff
@@ -85,14 +86,17 @@ public QueryResource(
       QuerySegmentWalker texasRanger,
       ServiceEmitter emitter,
       RequestLogger requestLogger,
-      QueryManager queryManager
+      QueryManager queryManager,
+      InjectableValues guiceInjectableValues
   )
   {
     this.config = config;
     this.jsonMapper = jsonMapper.copy();
+    this.jsonMapper.setInjectableValues(guiceInjectableValues); // Workaround for https://github.com/FasterXML/jackson-databind/issues/696
```

Comment:
This seems to be happening because we are storing a copy of the ObjectMapper, can we instead just not try to store a copy?

I *think* we are doing this so that we can set the config below this, but from looking at the code, I don't think we need to do that even.  If closing the OutputStream we are being handed in the streaming response breaks something, then we can wrap that object such that the close() method doesn't actually do anything instead of setting this parameter.